packages/amplify-schema-validator/src/validators/enum-is-defined-once.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  DocumentNode,
+  Kind,
+  EnumTypeDefinitionNode,
+} from 'graphql';
+import { ValidationError } from '../exceptions/validation-error';
+
+/**
+     * Enum is defined once
+     *
+     * @param schema graphql schema
+     * @returns true if a enum is defined once
+     */
```

Comment:
Somewhitespace weirdness on this comment

Reply:
Fixed